modular_skyrat/modules/customization/__DEFINES/DNA.dm

```diff
@@ -1,3 +1,7 @@
 #define DNA_BLOCKS_PER_FEATURE 4
+#define DNA_FEATURE_COLOR_BLOCKS_PER_FEATURE 3
+#define DNA_FEATURE_BLOCKS_TOTAL_SIZE_PER_FEATURE (DNA_BLOCK_SIZE+DNA_BLOCK_SIZE_COLOR*DNA_FEATURE_COLOR_BLOCKS_PER_FEATURE)
 #define DNA_BLOCKS_PER_MARKING 2
+#define DNA_MARKING_COLOR_BLOCKS_PER_MARKING 1
 #define DNA_BLOCKS_PER_MARKING_ZONE 1+MAXIMUM_MARKINGS_PER_LIMB*DNA_BLOCKS_PER_MARKING
+#define DNA_MARKING_BLOCKS_TOTAL_SIZE_PER_MARKING (DNA_BLOCK_SIZE+DNA_BLOCK_SIZE_COLOR*DNA_MARKING_COLOR_BLOCKS_PER_MARKING)
```

Comment:
Y0SH1, I really, really appreciate your help, but please put spaces around your symbols, thank you.
```suggestion
#define DNA_BLOCKS_PER_FEATURE 4
#define DNA_FEATURE_COLOR_BLOCKS_PER_FEATURE 3
#define DNA_FEATURE_BLOCKS_TOTAL_SIZE_PER_FEATURE (DNA_BLOCK_SIZE + DNA_BLOCK_SIZE_COLOR * DNA_FEATURE_COLOR_BLOCKS_PER_FEATURE)
#define DNA_BLOCKS_PER_MARKING 2
#define DNA_MARKING_COLOR_BLOCKS_PER_MARKING 1
#define DNA_BLOCKS_PER_MARKING_ZONE MAXIMUM_MARKINGS_PER_LIMB * DNA_BLOCKS_PER_MARKING + 1
#define DNA_MARKING_BLOCKS_TOTAL_SIZE_PER_MARKING (DNA_BLOCK_SIZE + DNA_BLOCK_SIZE_COLOR * DNA_MARKING_COLOR_BLOCKS_PER_MARKING)
```